lib/babel.js

```diff
@@ -74,8 +77,33 @@ if (inputSourceMap) {
 	options.inputSourceMap = JSON.parse(inputSourceMap.map);
 }
 
+// load up custom compilers
+process.argv.slice(3).forEach(function (arg) {
+	var match = /^--compilers=.+?:(.+)$/i.exec(arg);
```

Comment:
> That means, really only --require is needed until we add watch support ourselves.

I'd prefer `--require`. It's easier to reason about when writing your own test bootstrap code and still works with other require hooks.

Mocha's watcher does weird things when it reloads the tests so I've just been using nodemon to restart the process entirely. I'm not convinced watching specific extensions is even necessary.